worldedit-core/src/main/java/com/fastasyncworldedit/core/function/pattern/SurfaceRandomOffsetPattern.java

```diff
@@ -13,13 +13,17 @@
 
     private final Pattern pattern;
     private final int moves;
+    private final int minY;
+    private final int maxY;
 
     private final MutableBlockVector3 cur;
     private final MutableBlockVector3[] buffer;
     private final MutableBlockVector3[] allowed;
 
-    public SurfaceRandomOffsetPattern(Pattern pattern, int distance) {
+    public SurfaceRandomOffsetPattern(Pattern pattern, int distance, int minY, int maxY) {
         this.pattern = pattern;
+        this.minY = minY;
+        this.maxY = maxY;
         this.moves = Math.min(255, distance);
```

Comment:
I assume that has been addressed in big worlds? Looking on the diff below

Reply:
This isn't to do with world height.